src/conf.c

```diff
@@ -66,9 +66,10 @@
 #define PASSWORD "([^@]*)"
 #define IP "((([0-9]{1,3})\\.){3}[0-9]{1,3})"
 #define IPMASK "(" IP "(/" DIGIT "+)?)"
+#define IPV6_SCOPE "((%[^ \\t\\/]{1,16})?)"
```

Comment:
Should this be named `IPV6SCOPE` without an underscore to follow the pattern of `IPV6MASK`?

Reply:
fixed in 29a0bf960c6be32c83c766c6a6327241c5ea31e1